### PR TITLE
add Hibernate service

### PR DESCRIPTION
**Pull request by Robert-Michalski**

### CODE REVIEW

File: src/main/java/com/ormanager/client/service/Hibernate.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+import java.lang.reflect.Field;
+import java.sql.*;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.StringJoiner;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import com.ormanager.orm.annotation.*;
```
Comment (olhakad) [inline src/main/java/com/ormanager/client/service/Hibernate.java]: please add empty line after import

File: src/main/java/com/ormanager/client/service/Hibernate.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+public class Hibernate<T> {
```
Comment (olhakad) [inline src/main/java/com/ormanager/client/service/Hibernate.java]: Why this class called Hibernate?

File: src/main/java/com/ormanager/client/service/Hibernate.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    private Connection con;
+    private AtomicLong id = new AtomicLong(0L);
+    private int idIndex = 1;
```
Comment (olhakad) [inline src/main/java/com/ormanager/client/service/Hibernate.java]: why id index starts from 1, not 0?

File: src/main/java/com/ormanager/client/service/Hibernate.java [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+        StringJoiner joiner = new StringJoiner(",");
+        for (Field field : declaredFields) {
+            if (field.isAnnotationPresent(Id.class)) {
+                pk = field;
+            } else {
+                joiner.add(field.getName());
+                columns.add(field);
+            }
+        }
+        int length = columns.size()+1;
```
Comment (olhakad) [inline src/main/java/com/ormanager/client/service/Hibernate.java]: format your code

File: src/main/java/com/ormanager/client/service/Hibernate.java [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+        String qMarks = IntStream.range(0, length)
+                .mapToObj(e -> "?")
+                .collect(Collectors.joining(","));
+        String sql = "INSERT INTO " + clazz.getSimpleName() + "( " + pk.getName()+"," + joiner.toString() + ") " + "values ("+qMarks+")";
+        System.out.println(sql);
```
Comment (olhakad) [inline src/main/java/com/ormanager/client/service/Hibernate.java]: logger was added to project, don't use system out

File: src/main/java/com/ormanager/client/service/Hibernate.java (hunk 1 above)
Comment (Robert-Michalski) [reply src/main/java/com/ormanager/client/service/Hibernate.java]: done

**Review by olhakad — APPROVED** [review]

**Pull request merged**